File: Módulo Python/Códigos/2 - Estrutura Sequencial/Exercicio.25.py
#25. Faça um algoritmo que leia o valor dos catetos de um triângulo retângulo, calcule e mostre sua hipotenusa.

cateto1 = float(input('Digite o valor do primeiro cateto: '))
cateto2 = float(input('Digite o valor do segundo cateto: '))

hipotenusa = (cateto1**2 + cateto2**2)**0.5
print(f"A hipotenusa é: {hipotenusa}")

File: Livro - Módulo Java/2 - Estrutura Sequencial/Exercicio.25.java
// 25 - Faça um algoritmo que leia o valor dos catetos de um triângulo
// retângulo, calcule e mostre sua hipotenusa.

import java.util.Scanner;

public class HipotenusaTriangulo {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor do primeiro cateto: ");
        double cateto1 = scanner.nextDouble();

        System.out.print("Digite o valor do segundo cateto: ");
        double cateto2 = scanner.nextDouble();

        double hipotenusa = Math.sqrt(Math.pow(cateto1, 2) + Math.pow(cateto2, 2));

        System.out.println("A hipotenusa do triângulo é: " + hipotenusa);

        scanner.close();
    }
}